File: backend/member/src/main/java/com/paymong/member/things/repository/ThingsRepository.java
package com.paymong.member.things.repository;


import com.paymong.member.things.dto.response.FindThingsListResDto;
import com.paymong.member.things.entity.Things;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;
import java.util.Optional;

public interface ThingsRepository extends JpaRepository<Things, Long> {

    List<Things> findAllByMemberId(Long memberId);

    void deleteByMemberIdAndThingsId(Long memberId, Long thingsId);

    Optional<Things> findByMemberIdAndRoutine(Long memberId, String routine);
}

File: android/data/src/main/java/com/paymong/data/repository/ThingsRepository.kt
package com.paymong.data.repository

import androidx.lifecycle.MutableLiveData

class ThingsRepository {

    private var _thingsCode: MutableLiveData<String> = MutableLiveData<String>()
    val thingsCode: MutableLiveData<String> = _thingsCode

    fun changeThingsCode(thingsCode: String) {
        _thingsCode.postValue(thingsCode)
    }

}